src/qililab/platform/platform.py

```diff
@@ -727,10 +734,12 @@ def _execute_qprogram_with_quantum_machines(  # pylint: disable=too-many-locals
         acquisitions = cluster.get_acquisitions(job=job)
 
         results = QProgramResults()
+        # Doing manual classification of results as QM does not return thresholded values like Qblox
         for measurement in measurements:
             measurement_result = QuantumMachinesMeasurementResult(
-                *[acquisitions[handle] for handle in measurement.result_handles]
+                *[acquisitions[handle] for handle in measurement.result_handles],
             )
+            measurement_result.set_classification_threshold(thresholds.get(measurement.bus, None))
```

Comment:
Is there a default for this thesholds or is it a mandatory field and whenever we do the measurment without the threshold in the runcard it will crash, if that is the case please create an exception to execute an error

Reply:
It is not required in the runcard, when we initialize the QM measurement result class we initialize this property to `None`. Then when we ask for the thresholded results if the `threshold` is still None then an array of zeros is returned in order to keep consistency with `Qblox`. But it is true that maybe I could raise a warning and document it better on the docstrings